CONTRIBUTING.md

```diff
@@ -260,6 +260,19 @@ For managing includes in C files, use [include-what-you-use].
 
 See [#549][549] for more details.
 
+### Lua Scripting
+
+Neovim uses precompiled Lua bytecode, so Lua files in [`./runtime`](./runtime) won't get used if you run a newly-built Neovim.
```

Comment:
This is not true for all lua files in runtime. While this list is subject to change, and thus the _general advice_ to use `--luamod-dev` here is the correct one, this paragraph also suggests that the absence of this flag causes no `runtime/lua` files to be used, which is not the case.

Reply:
Thanks for the precision 👍 
How about:

```suggestion
Neovim uses precompiled Lua bytecode, some Lua files in [`./runtime`](./runtime) won't get used if you run a newly-built Neovim.
```

I'd be better to have the list/reference to the list, but I can see that it would be confusing as well.